apollo-ios-codegen/Sources/IR/IR+NamedFragmentSpread.swift

```diff
@@ -18,7 +18,7 @@ public class NamedFragmentSpread: Hashable, CustomDebugStringConvertible {
   /// Indicates the location where the fragment has been "spread into" its enclosing
   /// operation/fragment. It's `scopePath` and `entity` reference are scoped to the operation it
   /// belongs to.
-  let typeInfo: SelectionSet.TypeInfo
+  public let typeInfo: SelectionSet.TypeInfo
```

Comment:
I don't think there is any reason this shouldn't be a public constant. Doing so matches what we make available on `InlineFragment` and means we don't need to have a `public` method just to get at `deferCondition`. The type info is also used in the deferred fragment metadata template, but that may change.